src/ophys_etl/modules/median_filtered_max_projection/utils.py

```diff
@@ -0,0 +1,213 @@
+import pathlib
+import h5py
+import numpy as np
+import multiprocessing
+import multiprocessing.managers
+import scipy.ndimage as scipy_ndimage
+from ophys_etl.utils.array_utils import (
+    downsample_array,
+    n_frames_from_hz)
+
+
+def apply_median_filter_to_video(
```

Comment:
This appears private to this module. Could you please mark it as private?

Reply:
I don't see it as private. For the moment it is only used here, but it is an operation ("apply a median filter to every frame in a video") that I could imagine us wanting to use in future data processing pipelines. I'm going to leave it as-is.